cirq/sim/density_matrix_simulator.py

```diff
@@ -30,6 +30,14 @@
     from typing import Any, Hashable
 
 
+class _StateAndBuffers:
+
+    def __init__(self, n: int, state: np.ndarray):
```

Comment:
num_qubits?  or add doc about what n is.

Reply:
Done.